app.js

```diff
@@ -0,0 +1,16 @@
+const hamburgerOpenConst = document.querySelector('.hamburgerOpen');
+const hamburgerCloseConst = document.querySelector('.hamburgerClose');
+const menuPageConst = document.querySelector('.menuPage');
+// const pageLinkConst = document.querySelectorAll('.pageLink');
+
```

Comment:
- [ ] following the best practices, you are not meant to leave unused code as a comment in your code base, it makes it look unprofessional, I'll suggest you remove this. :wave: